### PR TITLE
made move and copy overlay in content-bundle responsive

### PR DESCRIPTION
**Pull request by nnatter**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | -
| Related issues/PRs | https://github.com/massiveart/husky/pull/703
| License | MIT
| Documentation PR | -

#### What's in this PR?

This PR makes the width of the move- and copy-overlay of the content bundle responsive (similar to the media-selection-overlay in the document-tab of the contact bundle).
Additionally some dead code in the column-component of the content-bundle is removed.

### CODE REVIEW

**Comment by nnatter** [comment]
@danrot

File: src/Sulu/Bundle/ContentBundle/Resources/public/scss/modules/column-navigation-overlay.scss [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    @media(min-width: 1600px) {
+        &, .slide, .overlay-footer {
+            width: 1360px !important;
+        }
+    }
+
+    .overlay-content {
+        margin: 0px;
+    }
+}
```
Comment (danrot) [inline src/Sulu/Bundle/ContentBundle/Resources/public/scss/modules/column-navigation-overlay.scss]: Add a new line

File: src/Sulu/Bundle/ContentBundle/Resources/public/scss/modules/column-navigation-overlay.scss [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+    @media(max-width: 1024px) {
+        &, .slide, .overlay-footer {
+            width: 720px !important;
+        }
+    }
+
+    @media(min-width: 1600px) {
+        &, .slide, .overlay-footer {
+            width: 1360px !important;
```
Comment (danrot) [inline src/Sulu/Bundle/ContentBundle/Resources/public/scss/modules/column-navigation-overlay.scss]: Are all the `!important`s  in this file really necessary?

**Comment by danrot** [comment]
Looks good, please rebase and it will be the first PR I merge tomorrow morning 😉

**Comment by danrot** [comment]
Just recognized: This should probably also be applied for the internal links content type, shouldn't it?

**Comment by nnatter** [comment]
@danrot: internal-links and single-internal-link overlay are responsive now too.

**Pull request merged**